classes/Hook.php

```diff
@@ -89,19 +91,46 @@ public function add($autodate = true, $null_values = false)
         return parent::add($autodate, $null_values);
     }
 
+    public static function isDisplayHookName($hook_name)
```

Comment:
@alexsegura Thanks, stole your function :)
The lowercase 'displayheader' is probably just some legacy thing...